### DISCUSSION TITLE
How can I search for news within my hub using PnP Modern Search box?

### DISCUSSION THREAD

**Discussion by khha** [microsoft-search/pnp-modern-search#4002] [General]
I am using SharePoint Online and pnp search box webpart. I would like to use the OOTB search result page. Which querystring can I add to the url to show only news from the associated sites to my hub site?

I try this but it is not working:

https://myCompany.sharepoint.com/_layouts/15/search.aspx/sitenews?q=testPageTitle&DepartmentId=dc0cbdbb-28b8-4d30-b46d-8bd0b24559bd

https://myCompany.sharepoint.com/_layouts/15/search.aspx/sitenews?q=testPageTitle&scope=hub

**Comment by wobba** [comment]
The URL's you list are for out of the box search in SharePoint, not pertaining these web parts. Using the pnp search box with the out of the box results page is not a highly intended scenario.

When you link to https://mycompany.sharepoint.com/_layouts/15/search.aspx/sitenews that is a site scoped page which would limit results to that sitecollection only. Not allowing you to bring in results from other sitecollections.

**Reply by khha** [reply]
> The URL's you list are for out of the box search in SharePoint, not pertaining these web parts. Using the pnp search box with the out of the box results page is not a highly intended scenario.
> 
> When you link to https://mycompany.sharepoint.com/_layouts/15/search.aspx/sitenews that is a site scoped page which would limit results to that sitecollection only. Not allowing you to bring in results from other sitecollections.

@wobba thank you, Is it true that when I create a new page and add the pnp search result webpart I can reach my goal? 

For example this is my created page with the pnp search result webpart https://myCompany.sharepoint.com/SitePages/intranet-hubsite-news.aspx.

On the welcome page of my intranet I have a pnp searchbox webpart which sends the search querystring to my custom pnp search result page. And in the pnp search result webpart I need to put the filter DepartmentId=dc0cbdbb-28b8-4d30-b46d-8bd0b24559bd .

![image](https://github.com/user-attachments/assets/7d4713d2-ac96-4af8-90f6-d3c29a7b0c76)

**Reply by wobba** [reply]
You typically add a PnP Search Result web part together with the search box web part. There are tutorials at https://microsoft-search.github.io/pnp-modern-search/scenarios/
